compiler/ztests/par-count.yaml

```
@@ -2,32 +2,40 @@ script: |
   export ZED_LAKE=test
   zed init -q
   zed create -q -orderby ts test
-  zc -C -P 2 "from test | count() by y" | sed -e 's/pool .* =>/pool POOL =>/'
+  zc -C -P 2 "from test | count() by y" | sed -e 's/lister pool.*/lister .../' -e 's/seqscan pool .*/seqscan .../'
```

Nit: Simplify all these sed commands.
```suggestion
  zc -C -P 2 "from test | count() by y" | sed -e 's/pool .*$/.../'
```